src/test/groovy/graphql/schema/fetching/ConfusedPojo.java

```diff
@@ -0,0 +1,12 @@
+package graphql.schema.fetching;
+
+public class ConfusedPojo {
+
+    public String getRecordLike() {
+        return "getRecordLike";
+    }
+
+    public String recordLike() {
+        return "recordLike";
+    }
```

Comment:
IN theory this is possible - stupid but possible - in the old days the getter would be found so this shows that this still happens.  eg record getters are looked up after pojo getters